inference/models/falcon.cc

```diff
@@ -183,9 +183,9 @@ void FALCON::create_falcon_model(FFModel &ff,
   Tensor output;
   if (mode == BEAM_SEARCH_MODE) {
     Tensor softmax = ff.softmax(lm_head, -1);
-    output = ff.beam_top_k(softmax, falcon_config.max_beam_width, false);
+    output = ff.argmax(softmax, /*beam_Search*/ true);
```

Comment:
instead of modifying the model, as a temporary fix, why not put the new code in a `#if defined(FF_USE_HIP_ROCM)` block, and the old code in a `#if defined(FF_USE_CUDA)`. This way we don't risk forgetting about this

Reply:
This is a legacy issue, all other models are using argmax, I forgot to update this one before.

Reply:
ok, my bad!